buttons/src/test/kotlin/com/decathlon/vitamin/compose/buttons/VitaminButtonsPrimaryTest.kt

```diff
@@ -42,7 +40,15 @@ class VitaminButtonsPrimaryTest(
                         verticalArrangement = Arrangement.spacedBy(8.dp)
                     ) {
                         Text(text = variant.name, style = VitaminTheme.typography.subtitle1)
-                        Row(horizontalArrangement = Arrangement.spacedBy(4.dp)) {
+                        val rowModifier = if (variant == Variant.GhostReversed) {
+                            Modifier.background(VitaminTheme.colors.vtmnBackgroundBrandPrimary)
```

Comment:
At the first read of the new screenshots, I didn't understand why you changed the background of ghost reversed buttons before reading this line.

Rather than using background brand primary, maybe we can use surface color which seems more appropriated? What do you think about it?